src/components/table/columns.ts

```diff
@@ -24,6 +24,7 @@ export class ColumnDefinitionFactory {
             field: column.field,
             hozAlign: column.horizontalAlign,
             headerSort: column.headerSort,
+            headerSortStartingDir: 'desc',
```

Comment:
Do we want to change the default in lime-elements without providing a way to change it? Also seems like a breaking change 🤔 

I think we rather want to add `headerSortStartingDir` to our `Column` type and set that instead (possibly depending on the field type) when passing the `columns` prop the the table component.

Or maybe call it `headerSortDescending: boolean` instead 🙂

Reply:
I agree, in order to implement it for specific field types we'd have to do that. I'll give that approach a try. 😃 

This first approach won't set the default sorting on columns to descending when the table is loaded, it simply ensures that descending is the first direction the tabulator sets on a column when the user clicks on it to sort it.

Reply:
> I'm not sure if this approach is a breaking changes as it won't set the default sorting on columns to descending when the table is loaded, it simply ensures that descending is the first direction the tabulator sets on a column when the user clicks on it to sort it.

You're right. It's probably not a big deal to change that default behaviour. 🙂